examples/Notebooks/Performance/WSCC_9bus_mult_coupled_validation.ipynb

```diff
@@ -0,0 +1,196 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d38ae51f-7dcc-4d8f-91bf-3f0a897e0e45",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import villas.dataprocessing.readtools as rt\n",
+    "from villas.dataprocessing.timeseries import TimeSeries as ts\n",
+    "import matplotlib.pyplot as plt\n",
+    "import re\n",
+    "import numpy as np\n",
+    "import math\n",
+    "import os\n",
+    "import dpsimpy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "534e7ab2-2784-491f-aa49-132fb89e4f29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import glob\n",
+    "\n",
+    "def download_grid_data(name, url):\n",
+    "    with open(name, 'wb') as out_file:\n",
+    "        content = requests.get(url, stream=True).content\n",
+    "        out_file.write(content)\n",
+    "\n",
+    "url = 'https://raw.githubusercontent.com/dpsim-simulator/cim-grid-data/master/WSCC-09/WSCC-09_RX/WSCC-09_RX'\n",
+    "filename = 'WSCC-09_RX'\n",
+    "download_grid_data(filename+'_EQ.xml', url+'_EQ.xml')\n",
+    "download_grid_data(filename+'_TP.xml', url+'_TP.xml')\n",
+    "download_grid_data(filename+'_SV.xml', url+'_SV.xml')\n",
+    "download_grid_data(filename+'_DI.xml', url+'_DI.xml')\n",
+    "\n",
+    "files = glob.glob(filename+'_*.xml')\n",
+    "print(files)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77f5bb79-6009-4b4d-afbb-357927b965a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "final_time = 0.5\n",
+    "time_step = 1e-4\n",
+    "\n",
+    "log_dir = \"logs\"\n",
+    "log_level = dpsimpy.LogLevel.info\n",
+    "\n",
+    "num_copies = 0\n",
+    "num_threads = 0\n",
+    "num_seq = 0\n",
+    "sim_name = \"WSCC_9bus_coupled_\" + str(num_copies) + \"_\" + str(num_threads) + \"_\" + str(num_seq)\n",
+    "\n",
+    "# functions are missing! FIXME\n",
```

Comment:
Can be cleaned up here I guess, as scalability is not of relevance here for validation.

Reply:
done